src/components/space/charts/MemberActionsChart.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import { Typography } from '@mui/material';
+
+import { Action } from '@graasp/sdk';
+
+import { endOfWeek, format } from 'date-fns';
+import { groupBy } from 'lodash';
+import {
+  Bar,
+  CartesianGrid,
+  ComposedChart,
+  Legend,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+
+import ChartContainer from '@/components/common/ChartContainer';
+import { getColorForActionTriggerType } from '@/config/constants';
+import { useActionsTranslation, useAnalyticsTranslation } from '@/config/i18n';
+
+type Props = {
+  actionsGroupedByWeekStart: { [key: string]: Action[] };
+};
+const MemberActionsChart = ({
+  actionsGroupedByWeekStart,
+}: Props): JSX.Element => {
+  const { t } = useAnalyticsTranslation();
+  const { t: translateAction } = useActionsTranslation();
+
+  const types: string[] = [];
```

Comment:
Where I supposed you get `actions` instead of them being grouped from the beginning. 
```suggestion
  const types = groupBy(actions, 'type');
```